pkg/gateway/model_build_targetgroup_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/aws/aws-application-networking-k8s/pkg/utils/gwlog"
```

Comment:
please add imports sort and grouping, usually order std lib, 3ed party, local

 https://www.jetbrains.com/help/go/creating-and-optimizing-imports.html#use-the-goimports-local-grouping

Reply:
Done next commit, thanks for the jetbrains link!